src/js/htmlutils.js

```diff
@@ -143,11 +145,16 @@ var htmlUtils = exports.htmlUtils = {
 
     // If origin has been whitelisted set text for DNT.
     var whitelistedText = '';
+    var actionDescription = '';
     if (isWhitelisted) {
       whitelistedText = '' +
         '<div id="dnt-compliant">' +
         '<a target=_blank href="https://www.eff.org/privacybadger#faq--I-am-an-online-advertising-/-tracking-company.--How-do-I-stop-Privacy-Badger-from-blocking-me?">' +
-        '<img src="/icons/dnt-16.png" title="This domain promises not to track you."></a></div>';
+        '<img src="/icons/dnt-16.png"></a></div>';
+
+      actionDescription = i18n.getMessage('dnt_tooltip');
```

Comment:
If getting the description for DNT domains is part of `getActionDescription`, it will also benefit from the caching introduced in https://github.com/EFForg/privacybadger/commit/7fddc74cba7d5e5d8721c4bfb2947a726d7092c2 for example.